starr/modules/tags.py

```diff
@@ -266,8 +261,7 @@ async def tag_edit_command(
 
             if event.interaction.custom_id == "yes":
                 await bot.db.execute(
-                    "INSERT INTO tags (GuildID, TagOwner, TagName, TagContent) "
-                    "VALUES ($1, $2, $3, $4);",
+                    "INSERT INTO tags (GuildID, TagOwner, TagName, TagContent) VALUES ($1, $2, $3, $4);",
```

Comment:
Same as above this needs to go to 2 lines because of len8.
```suggestion
                    "INSERT INTO tags (GuildID, TagOwner, TagName, TagContent) "
                    "VALUES ($1, $2, $3, $4);",
```